.github/workflows/deploy-to-node.yaml

```diff
@@ -33,6 +33,14 @@ jobs:
       WEB_EMAIL_SERVER_PORT: ${{ secrets.DEV_WEB_EMAIL_SERVER_PORT }}
       WEB_EMAIL_SERVER_USER: ${{ secrets.DEV_WEB_EMAIL_SERVER_USER }}
       WEB_NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
+      WEB_NEXT_PUBLIC_CLOUDFLARE_CAPTCHA_SITE_KEY:
+        ${{ secrets.DEV_WEB_NEXT_PUBLIC_CLOUDFLARE_CAPTCHA_SITE_KEY }}
+      WEB_CLOUDFLARE_CAPTCHA_SERCERT_KEY:
+        ${{ secrets.DEV_WEB_CLOUDFLARE_CAPTCHA_SERCERT_KEY }}
```

Comment:
@fozziethebeat thx for preparing.  I have pushed these changes now already because the captcha settings were missing. see https://github.com/LAION-AI/Open-Assistant/commit/776584503f6b7d0fd18ddcb48bcea6ded4119543